### ISSUE TITLE
Analysis of target '//tensorflow/tools/pip_package:build_pip_package' failed; build aborted: no such package '@local_config_cc//'

### ISSUE THREAD

**Issue by jeffsaremi** [tensorflow/tensorflow#28047]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): source
- TensorFlow version: version r1.14
- Python version: 3.6
- Installed using virtualenv? pip? conda?: Conda
- Bazel version (if compiling from source): 0.24.1
- GCC/Compiler version (if compiling from source): VisualStudio 2017
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

Trying to build tensorflow.
Have installed bazel and MySys2 and python
Configured for no GPU (see below for print outs)
Issuing the following command results in error:
`bazel build --config=monolithic //tensorflow/tools/pip_package:build_pip_package`

**CONFIGURE OUTPUT**
```
(tf3.6) D:\repos\tensorflow>python ./configure.py
Extracting Bazel installation...
WARNING: --batch mode is deprecated. Please instead explicitly shut down your Bazel server using the command "bazel shutdown".
You have bazel 0.24.1 installed.
Please specify the location of python. [Default is C:\Users\jesaremi\AppData\Local\Continuum\anaconda3\envs\tf3.6\python.exe]:


Found possible Python library paths:
  C:\Users\jesaremi\AppData\Local\Continuum\anaconda3\envs\tf3.6\lib\site-packages
Please input the desired Python library path to use.  Default is [C:\Users\jesaremi\AppData\Local\Continuum\anaconda3\envs\tf3.6\lib\site-packages]

Do you wish to build TensorFlow with XLA JIT support? [y/N]:
No XLA JIT support will be enabled for TensorFlow.

Do you wish to build TensorFlow with ROCm support? [y/N]:
No ROCm support will be enabled for TensorFlow.

Do you wish to build TensorFlow with CUDA support? [y/N]:
No CUDA support will be enabled for TensorFlow.

Please specify optimization flags to use during compilation when bazel option "--config=opt" is specified [Default is /arch:AVX]:


Would you like to override eigen strong inline for some C++ compilation to reduce the compilation time? [Y/n]:
Eigen strong inline overridden.

Preconfigured Bazel build configs. You can use any of the below by adding "--config=<>" to your build command. See .bazelrc for more details.
        --config=mkl            # Build with MKL support.
        --config=monolithic     # Config for mostly static monolithic build.
        --config=gdr            # Build with GDR support.
        --config=verbs          # Build with libverbs support.
        --config=ngraph         # Build with Intel nGraph support.
        --config=numa           # Build with NUMA support.
        --config=dynamic_kernels        # (Experimental) Build kernels into separate shared objects.
Preconfigured Bazel build configs to DISABLE default on features:
        --config=noaws          # Disable AWS S3 filesystem support.
        --config=nogcp          # Disable GCP support.
        --config=nohdfs         # Disable HDFS support.
        --config=noignite       # Disable Apache Ignite support.
        --config=nokafka        # Disable Apache Kafka support.
        --config=nonccl         # Disable NVIDIA NCCL support.


```

**BUILD OUTPUT**
```
(tf3.6) D:\repos\tensorflow>bazel build --config=monolithic //tensorflow/tools/pip_package:build_pip_package
Starting local Bazel server and connecting to it...
WARNING: The following configs were expanded more than once: [monolithic]. For repeatable flags, repeats are counted twice and may lead to unexpected behavior.
ERROR: no such package '@local_config_cc//': Traceback (most recent call last):
        File "C:/users/jesaremi/_bazel_jesaremi/fh3jreae/external/bazel_tools/tools/cpp/cc_configure.bzl", line 52
                configure_windows_toolchain(repository_ctx)
        File "C:/users/jesaremi/_bazel_jesaremi/fh3jreae/external/bazel_tools/tools/cpp/windows_cc_configure.bzl", line 406, in configure_windows_toolchain
                setup_vc_env_vars(repository_ctx, vc_path)
        File "C:/users/jesaremi/_bazel_jesaremi/fh3jreae/external/bazel_tools/tools/cpp/windows_cc_configure.bzl", line 204, in setup_vc_env_vars
                execute(repository_ctx, ["./get_env.bat"], e...)
        File "C:/users/jesaremi/_bazel_jesaremi/fh3jreae/external/bazel_tools/tools/cpp/lib_cc_configure.bzl", line 163, in execute
                auto_configure_fail(("non-zero exit code: %d, comman...)))
        File "C:/users/jesaremi/_bazel_jesaremi/fh3jreae/external/bazel_tools/tools/cpp/lib_cc_configure.bzl", line 109, in auto_configure_fail
                fail(("\n%sAuto-Configuration Error:%...)))

Auto-Configuration Error: non-zero exit code: 255, command ["./get_env.bat"], stderr: (";D:\Program Files\CMake\bin;C:\Program Files\Java\jdk1.8.0_201\bin;D:\Program Files\apache-maven-3.3.9\bin;D:\Program Files\protoc\bin;D:\Program Files\scala-2.12.8\bin;C:\msys64\usr\bin;C:\Program Files (x86)\Notepad++;D:\Program Files\bazel;"=="" was unexpected at this time.
)
```

**Comment by meteorcloudy** [comment]
@jeffsaremi Can you do a 
```
cat bazel-tensorflow/external/local_config_cc/get_env.bat
```
and tell me what's the content of this file?

**Comment by jeffsaremi** [comment]
@meteorcloudy :

```
(tf3.6) D:\repos\tensorflow>cat bazel-tensorflow/external/local_config_cc/get_env.bat
@echo off
call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\\VC\Auxiliary\Build\VCVARSALL.BAT" amd64 > NUL
echo PATH=%PATH%,INCLUDE=%INCLUDE%,LIB=%LIB%,WINDOWSSDKDIR=%WINDOWSSDKDIR%

```

**Comment by meteorcloudy** [comment]
@jeffsaremi The file looks correct to me, can you run it in cmd.exe and see what's the output?

**Comment by jeffsaremi** [comment]
@meteorcloudy 
Is this what you asked for?
```
(tf3.6) D:\repos\tensorflow>"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\\VC\Auxiliary\Build\VCVARSALL.BAT" amd64
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.9.11
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

```

**Comment by jeffsaremi** [comment]
I think there is something wrong with the parameters I am choosing. For instance --config. I don't know why I chose monolithic but the documentation didn't really advise anything on this.

**Comment by meteorcloudy** [comment]
Almost, can you try to run `bazel-tensorflow\external\local_config_cc\get_env.bat` and see what happens? Because the following error looks like comes from running this script.
```
;D:\Program Files\CMake\bin;C:\Program Files\Java\jdk1.8.0_201\bin;D:\Program Files\apache-maven-3.3.9\bin;D:\Program Files\protoc\bin;D:\Program Files\scala-2.12.8\bin;C:\msys64\usr\bin;C:\Program Files (x86)\Notepad++;D:\Program Files\bazel;"=="" was unexpected at this time.
```

**Comment by jeffsaremi** [comment]
I ran that and got the same error complaining about the ==.
I then did this:
`set VSCMD_DEBUG=3`
to get debug prints from the `vcvarsall.bat `in `VisualStudio`
and upon examining the print outs I realized that I had a `"` (double quote) in my path, caused by having an entry which itself had a semicolon in it.
After fixing the path, I ran the build and is running with no problems.
We can close this case now.
